shared_model/validators/transactions_collection/transactions_collection_validator.hpp

```diff
@@ -24,6 +24,7 @@ namespace shared_model {
       TransactionValidator transaction_validator_;
       std::shared_ptr<AbstractValidator<interface::TransactionBatch>>
           batch_validator_;
+      bool duplicate_tolerance_;
```

Comment:
Not clear. Have to be renamed to `txs_duplicates_allowed_`.

Reply:
Fixed